examples/composite_template/ex_application_window/ex_application_window.ui

```diff
@@ -17,27 +17,125 @@
     <child>
       <object class="GtkBox">
         <property name="orientation">vertical</property>
-        <property name="valign">center</property>
-        <property name="spacing">6</property>
-        <property name="margin-start">12</property>
-        <property name="margin-end">12</property>
-        <property name="margin-top">12</property>
-        <property name="margin-bottom">12</property>
         <child>
```

Comment:
the diff is caused by a change in the indentation of this file; it used to use 2 spaces iirc. Maybe we should enforce that using an .editorconfig file

Reply:
It's still 2 spaces, the diff happened because this box got put a level deeper in the tree.